onnx/subbyte_helper.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Tuple, Union
+
+import numpy as np
+
+
+def float32_to_4bit_unpacked(
+    x: Union[np.ndarray, np.dtype, float], signed: bool
+) -> int:
```

Comment:
Is it possible for the function to return anything other than int? Looks like nparray is a possibility?